### PR TITLE
Cleanup RancherOS references

### PR DESCRIPTION
**Pull request by mudler**
Signed-off-by: Ettore Di Giacinto <edigiacinto@suse.com>

### CODE REVIEW

**Comment by mudler** [comment]
Supersedes https://github.com/rancher/elemental/pull/187

File: framework/files/usr/sbin/elemental-populate-node-labels [hunk 1]
```diff
@@ -6,10 +6,10 @@ set -e
 IMAGE=$(echo "$IMAGE" | sed 's![/:]!-!g')
 for i in rke2 k3s; do
     mkdir -p /etc/rancher/$i/config.yaml.d
-    cat > /etc/rancher/$i/config.yaml.d/99-rancheros-labels.yaml << EOF
+    cat > /etc/rancher/$i/config.yaml.d/99-elemental-labels.yaml << EOF
 node-label+:
-- rancheros.cattle.io/image=${IMAGE}
```
Comment (mudler) [inline framework/files/usr/sbin/elemental-populate-node-labels:11]: I hope this isn't relied by anything..

File: framework/files/usr/sbin/elemental-populate-node-labels [hunk 2]
```diff
@@ -6,10 +6,10 @@ set -e
...
-- rancheros.cattle.io/tag=${IMAGE_TAG}
+- elemental.cattle.io/image=${IMAGE}
+- elemental.cattle.io/tag=${IMAGE_TAG}
```
Comment (mudler) [inline framework/files/usr/sbin/elemental-populate-node-labels:12]: @kkaempf do we have the `IMAGE_TAG` in the os-release generated by OBS?

This file is sourcing `rancheros-release` file at the moment.

Comment (Itxaka) [reply framework/files/usr/sbin/elemental-populate-node-labels:12]: nope, but we can just move that to be on os-release with a small change so this should be good to go once we change that

**Comment by mudler** [comment]
Only things I see still missing with a grep now are, tests, old scripts used for building and docs. But that would likely be a better fit in a follow-up

**Review by Itxaka — APPROVED** [review]

**Pull request merged**